quota/noop.go

```diff
@@ -17,10 +17,23 @@ package quota
 import (
 	"context"
 	"fmt"
+
+	"github.com/golang/glog"
 )
 
+// NoopManagerName represents the noop quota implementation.
+const NoopManagerName = "noop"
```

Comment:
Does this need to be exported at this point? Nothing seems to be using it from outside this file.

Reply:
It might be helpful if someone wants to check whether the current quota system is "noop" though, and is consistent with the other quota implementations (which both have a `QuotaManagerName` constant).